deploy/docker-compose.yml

```diff
@@ -1,95 +1,79 @@
 services:
   ballerine-case-managment:
-    build:
-      context: ../apps/backoffice-v2/
-      args:
-        NPM_LOG_LEVEL: notice
+    image: ghcr.io/ballerine-io/backoffice:latest
+    platform: linux/amd64
     ports:
-      - ${BACKOFFICE_PORT}:80
+      - 5137:80
     depends_on:
       - ballerine-workflow-service
     restart: on-failure
-  ballerine-headless-example:
-    build:
-      context: ../examples/headless-example
-      args:
-        NPM_LOG_LEVEL: notice
+  ballerine-kyb-app:
+    image: ghcr.io/ballerine-io/kyb-app:latest
+    platform: linux/amd64
     ports:
-      - ${HEADLESS_SVC_PORT}:80
+      - 5201:80
     depends_on:
       - ballerine-workflow-service
     restart: on-failure
-  ballerine-kyb-app:
-    build:
-      context: ../apps/kyb-app
-      args:
-        NPM_LOG_LEVEL: notice
+    environment:
+      VITE_API_URL: 'http://localhost:3000/api/v1/'
+      VITE_KYB_DEFINITION_ID: 'kyb_parent_kyc_session_example'
+  ballerine-workflows-dashboard:
+    image: ghcr.io/ballerine-io/workflows-dashboard:latest
+    platform: linux/amd64
     ports:
-      - ${KYB_APP_PORT}:80
+      - 5200:80
     depends_on:
       - ballerine-workflow-service
     restart: on-failure
   ballerine-workflow-service:
-    build:
-      context: ../services/workflows-service/
-      args:
-        NPM_LOG_LEVEL: notice
+    image: ghcr.io/ballerine-io/workflows-service:latest
+    platform: linux/amd64
     command:
       - /bin/sh
       - -c
       - |
         npm run db:init
+        npm run seed
         dumb-init npm run prod
     ports:
-      - ${WORKFLOW_SVC_PORT}:${WORKFLOW_SVC_PORT}
+      - 3000:3000
     environment:
-      BCRYPT_SALT: ${BCRYPT_SALT}
-      DB_URL: postgres://${DB_USER}:${DB_PASSWORD}@db:${DB_PORT}
-      API_KEY: ${API_KEY}
-      NODE_ENV: ${NODE_ENV}
-      COMPOSE_PROJECT_NAME: ${COMPOSE_PROJECT_NAME}
-      DB_PORT: ${DB_PORT}
-      DB_USER: ${DB_USER}
-      DB_PASSWORD: ${DB_PASSWORD}
-      SESSION_SECRET: ${SESSION_SECRET}
-      SESSION_EXPIRATION_IN_MINUTES: ${SESSION_EXPIRATION_IN_MINUTES}
-      BACKOFFICE_CORS_ORIGIN: http://localhost:${BACKOFFICE_PORT}
-      WORKFLOW_DASHBOARD_CORS_ORIGIN: http://localhost:${WORKFLOW_DASHBOARD_PORT}
-      PORT: ${WORKFLOW_SVC_PORT}
-      ENVIRONMENT_NAME: ${ENVIRONMENT_NAME}
-      KYB_EXAMPLE_CORS_ORIGIN: http://localhost:${KYB_APP_PORT}
+      API_KEY: secret
+      BCRYPT_SALT: 10
+      BACKOFFICE_CORS_ORIGIN: http://localhost:5137
+      COMPOSE_PROJECT_NAME: ballerine-x
+      DB_URL: postgres://admin:password@ballerine-postgres:5432
+      DB_PORT: 5432
+      DB_USER: admin
+      DB_PASSWORD: password
+      NODE_ENV: development
+      PORT: 3000
+      SESSION_EXPIRATION_IN_MINUTES: 60
+      SESSION_SECRET: secret
+      WORKFLOW_DASHBOARD_CORS_ORIGIN: http://localhost:5200
+      KYB_EXAMPLE_CORS_ORIGIN: http://localhost:5201
+      IS_DEMO: true
+      WEBHOOK_SECRET: webhook_secret
+      APP_API_URL: https://alon.ballerine.dev
```

Comment:
im not sure if its by mistake or not, its seems not relevant for oss

Reply:
it would be better to define env our of docker compose as we usually did with the .env.example and if you would like to override any of the values you can just specify additional .env file for compose file like here:

```yaml
services:
  webapp:
    image: my-webapp-image
    env_file:
      - .env
      - .env.override
```